addons/mail/static/src/models/messaging_notification_handler/messaging_notification_handler.js

```diff
@@ -120,13 +120,43 @@ function factory(dependencies) {
                         last_message_id,
                         partner_id,
                     });
+                case 'deletion':
+                    return this._handleNotificationChannelDeletion(channelId, data);
                 case 'typing_status':
                     return this._handleNotificationChannelTypingStatus(channelId, data);
                 default:
                     return this._handleNotificationChannelMessage(channelId, data);
             }
         }
 
+        /**
+         * @private
+         * @param {integer} channelId
+         * @param {Object} param1
+         * @param {integer} param1.partner_id
+         * @param {string} param1.partner_name
+         */
+        async _handleNotificationChannelDeletion(channelId, { partner_id, partner_name }) {
+            const channel = this.env.models['mail.thread'].findFromIdentifyingData({
+                id: channelId,
+                model: 'mail.channel',
+            });
+            if (!channel) {
+                return;
+            }
+            channel.update({ isServerPinned: false });
+            partner_name = partner_id === this.messaging.currentPartner.id ? "You" : partner_name;
```

Comment:
the whole phrase needs to be translated differently, not just the pronoun, as it would change the conjugation